suite-native/intl/src/en.ts

```diff
@@ -399,6 +399,12 @@ export const en = {
                 updateButton: 'Update firmware',
                 title: 'Firmware update',
                 subtitle: 'New firmware is now available. Update your device now.',
+                changelog: {
+                    button: 'What’s new?',
+                    title: 'What’s new?',
+                    closeButton: 'Close',
```

Comment:
We've got a generic label for this:
`generic.buttons.close`

Reply:
fixed